ql/src/java/org/apache/hadoop/hive/ql/stats/StatsUtils.java

```diff
@@ -454,6 +468,26 @@ private static Statistics collectStatistics(HiveConf conf, PrunedPartitionList p
     return stats;
   }
 
+  private static Map<String, Long> getValidBasisStatsFromStorageHandler(Table table) {
```

Comment:
Seems like a typo in method name, Should it be `getValidBasicStatsFromStorageHandler`?

Reply:
Thanks @akshat0395 , i have updated the PR :) .